modules/programs/rtorrent.nix

```diff
@@ -1,18 +1,19 @@
 { config, lib, pkgs, ... }:
 
 with lib;
-
-let
-
-  cfg = config.programs.rtorrent;
-
+let cfg = config.programs.rtorrent;
 in {
   meta.maintainers = [ maintainers.marsam ];
 
+  imports = [
+    (mkRemovedOptionModule [ "programs" "rtorrent" "settings" ]
```

Comment:
As @sumnerevans mentioned, perhaps `mkRemovedOptionModule` instead of `mkRemovedOptionModule`?

Reply:
done